src/dmd/mtype.d

```diff
@@ -4922,6 +4922,28 @@ extern (C++) final class TypeFunction : TypeNext
         return MATCH.nomatch;
     }
 
+    /**
+     * Check if the arguments of a call are possibly inverted and prints a
+     * supplemental error message if the call would compile.
+     *
+     * Params:
+     *      tthis   = The type of the parent aggregate if it's a member function.
+     *      args    = The arguments of the call.
+     *      sc      = The scope of the call.
+     */
+    extern (D) void checkInvertedMatch(Type tthis, Expressions* args, Scope* sc)
+    {
+        if (args is null || args.length != 2)
+            return;
+        Expressions* invertedArgs = new Expressions(2);
```

Comment:
Can't `args` be passed to `callMatch` as is?

Reply:
No. When `checkInvertedMatch` is called it is known that the function cant be called as is so we look what does that gives if the params are passed in another order.

Reply:
You're swapping the order, right. My bad.